src/heretic/g_game.c

```diff
@@ -268,6 +268,14 @@ static int G_NextWeapon(int direction)
     return weapon_order_table[i].weapon_num;
 }
 
+// [crispy] holding down the "Run" key may trigger special behavior,
+// e.g. quick exit, clean screenshots, resurrection from savegames
```

Comment:
I thought I'd leave it in since I'll probably add similar behaviour for heretic to how it's used in Doom, and since it has the "may" qualifier. But ya, I can trim it for now and adjust later 😉